### ISSUE TITLE
[Bug]: iOS FlutterBluePlus.isAvailable throw error

### ISSUE THREAD

**Issue by lwd369** [boskokg/flutter_blue_plus#469]
### FlutterBluePlus Version

1.12.11

### Flutter Version

3.10.4

### What OS?

iOS

### OS Version

iOS 16.3.1

### What happened?

call `FlutterBluePlus.isAvailable` will throw `type 'int' is not a subtype of type 'FutureOr<bool>'`

### Logs

```shell
flutter: type 'int' is not a subtype of type 'FutureOr<bool>'
flutter: #0      FlutterBluePlus.isAvailable
<asynchronous suspension>
#1      BleService._checkBluetoothStatus
<asynchronous suspension>
#2      BleService.scanDevices
<asynchronous suspension>
#3      HomeController._scanCurrentDevice
<asynchronous suspension>
```

**Comment by chipweinberger** [comment]
regressed here https://github.com/boskokg/flutter_blue_plus/commit/a83ade69052b2d0794e1acc74fb85d12a0da07d3

**Comment by chipweinberger** [comment]
fixed in 1.12.13
